pinc/forum_interface.inc

```diff
@@ -335,12 +341,14 @@ function get_forum_user_details($username)
 
     $query = sprintf("
         SELECT *
-        FROM {$phpbb_table_prefix}_profile_fields_data
-        WHERE user_id = %d
-        ", $row["user_id"]);
+        FROM %s_profile_fields_data
+        WHERE user_id = %d",
+        $phpbb_table_prefix,
+        $row["user_id"]
+    );
 
     // This will fail on phpBB < 3.1, which is fine and expected
-    $res = mysqli_query(DPDatabase::get_connection(), $query);
+    $res = DPDatabase::query($query, FALSE);
```

Comment:
why FALSE?

Reply:
I'm basing it on the comment above ("this will fail on phpBB <3.1"). It appears the column names are different in older versions of phpBB, so the query will fail, and we don't want this to throw an exception.